INSTALL.md

```diff
@@ -3,6 +3,12 @@
 ## OpenStreetMap data
 You need OpenStreetMap data loaded into a PostGIS database (see below for [dependencies](#dependencies)). These stylesheets currently work only with the osm2pgsql defaults (i.e. database name is `gis`, table names are `planet_osm_point`, etc).
 
+Before loading the data, install the hstore extension into the database:
+
+```
+psql -d gis -c "CREATE EXTENSION hstore;"
+```
+
 It's probably easiest to grab an PBF of OSM data from [Mapzen](https://mapzen.com/metro-extracts/) or [geofabrik](http://download.geofabrik.de/). Once you've set up your PostGIS database, import with osm2pgsql:
```

Comment:
I also don't understand. IMO installing extensions is part of setting up the database, so importing is just what follows this step.